src/main/java/com/hyperwallet/clientsdk/model/VerificationDocument.java

```diff
@@ -0,0 +1,81 @@
+package com.hyperwallet.clientsdk.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@JsonInclude(Include.NON_NULL)
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+public class VerificationDocument {
+
```

Comment:
Pls rename this class to HyperwalletDocumentVerification

Reply:
Done.